huawei_lte_api/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.8.1'
+__version__ = '1.8.2'
```

Comment:
Please do not bump version number, that is done automatically by me on new version release using https://github.com/Salamek/version script that handles version change in all files...